Lib/ufo2ft/filters/__main__.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import importlib
+import logging
+
+import defcon
+from fontTools.misc.cliTools import makeOutputFileName
+
+from ufo2ft.filters import getFilterClass, logger
+
+logging.basicConfig(level=logging.INFO)
+
+parser = argparse.ArgumentParser(description="Filter a UFO file")
+parser.add_argument("--output", "-o", metavar="OUTPUT", help="output file name")
+parser.add_argument(
+    "--include", metavar="GLYPHS", help="comma-separated list of glyphs to filter"
+)
+parser.add_argument(
+    "--exclude", metavar="GLYPHS", help="comma-separated list of glyphs to not filter"
+)
+parser.add_argument("ufo", metavar="UFO", help="UFO file")
+parser.add_argument("filters", metavar="FILTER", nargs="+", help="filter name")
+
+args = parser.parse_args()
+if not args.output:
+    args.output = makeOutputFileName(args.ufo)
+
+ufo = defcon.Font(args.ufo)
```

Comment:
do you mind switching to ufoLib2 instead? or try ufoLib2 then defcon then ImportError